generators/constants.js

```diff
@@ -1,10 +1,10 @@
 const JAVA_VERSION = '11';
-const SPRING_BOOT_VERSION = '2.3.7.RELEASE';
-const SPRING_CLOUD_VERSION = 'Hoxton.SR9';
-const SPRING_DEP_MNGMNT_VERSION = '1.0.10.RELEASE';
+const SPRING_BOOT_VERSION = '2.4.2';
```

Comment:
The reason SpringBoot version is not updated to latest 2.4.2 is it is not yet supporting Spring Cloud AWS modules. We have Localstack autoconfiguration support which would break if we set SpringBoot version to 2.4.2.

Reply:
Reverted the version update